### PR TITLE
SPM-1695: remove account_number usage

### PR DESCRIPTION
**Pull request by psegedy**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: base/mqueue/platform_event.go [hunk 1]
```diff
@@ -13,38 +13,31 @@ type PlatformEvent struct {
 	ID          string                  `json:"id"`
 	Type        *string                 `json:"type"`
 	Timestamp   *types.Rfc3339Timestamp `json:"timestamp"`
-	Account     *string                 `json:"account"`
 	AccountID   int                     `json:"account_id"`
```
Comment (MichaelMraka) [inline base/mqueue/platform_event.go:16]: What is AccountID and how does it play with OrgID?

File: base/mqueue/platform_event.go [hunk 2]
```diff
@@ -13,38 +13,31 @@ type PlatformEvent struct {
...
 	OrgID       *string                 `json:"org_id,omitempty"`
 	B64Identity *string                 `json:"b64_identity"`
 	URL         *string                 `json:"url"`
 	SystemIDs   []string                `json:"system_ids,omitempty"`
 	RequestIDs  []string                `json:"request_ids,omitempty"`
 }
-
```
Comment (MichaelMraka) [inline base/mqueue/platform_event.go:24]: Accidently removed empty line.

File: base/mqueue/platform_event.go (hunk 1 above)
Comment (psegedy) [reply base/mqueue/platform_event.go:16]: I'm not completely sure, but based on the code `AccountID` is not actually coming from inventory but it is used internally for queries using `rh_account_id` column

Comment (MichaelMraka) [reply base/mqueue/platform_event.go:16]: I see, it's internal ID. We should probably name it better.

**Review by MichaelMraka — APPROVED** [review]

**Pull request merged**